### DISCUSSION TITLE
I am not able to avail benefits of the student developer pack of github.

### DISCUSSION THREAD

**Discussion by Akshta-19** [community/community#137708] [GitHub Education]
### Hi everyone,

I am not able to avail benefits of the student developer pack of github even after my profile has already been approved . What should i do?

**Comment by davevad93** [comment]
Hi @Akshta-19 ,

Don't worry you don't have to do anything, just wait, when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Akshta-19 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
